verification/curator-service/api/src/model/schedule.ts

```diff
@@ -6,8 +6,16 @@ export const scheduleSchema = new mongoose.Schema({
         required: 'Enter a CloudWatch schedule rule AWS Lambda ARN',
         match: [/^arn\:aws\:events\:.+\:.+\:rule\/.+/],
     },
+    awsScheduleExpression: {
+        type: String,
+        required: 'Enter a CloudWatch rule schedule expression',
+        match: [
+            /^((rate\(((1 (minute|hour|day))|([1-9]+ (minutes|hours|days)))\))|(cron\((.+\s){5}.+\)))/,
```

Comment:
what a delicious regex. the comment is v. helpful :D

the match property is cool!

Reply:
Yeah -- I considered getting more precise with the `cron([...])` spec, but decided this was the right balance for now. Not that I don't love a good write-only regex.